EventsExpress/ClientApp/src/store/initialState.js

```diff
@@ -75,6 +75,7 @@ const initialState = {
         isPending: true,
         isError: false,
         editedUser: null,
+        filter: null,
```

Comment:
Please don't name it so generic. If it is only for filter on users screen - please name it accordingly.